compiler/rustc_hir_typeck/src/fn_ctxt/checks.rs

```diff
@@ -1098,7 +1153,12 @@ impl<'a, 'tcx> FnCtxt<'a, 'tcx> {
                 Some(format!("provide the argument{}", if plural { "s" } else { "" }))
             }
             SuggestionText::Remove(plural) => {
-                Some(format!("remove the extra argument{}", if plural { "s" } else { "" }))
+                err.multipart_suggestion_verbose(
```

Comment:
I tend to prefer the verbose output in general, because I feel showing the resulting code ends up being clearer, but I can change this to terse, it shouldn't be too much of an issue.

Reply:
Changed. It's on its own commit so that you can review it.